src/cpp/session/modules/SessionCpp.R

```diff
@@ -0,0 +1,14 @@
+#
+# SessionCpp.R
+#
+# Copyright (C) 2022 by RStudio, PBC
+#
+# Unless you have received this program directly from RStudio pursuant
+# to the terms of a commercial license agreement with RStudio, then
+# this program is licensed to you under the terms of version 3 of the
+# GNU Affero General Public License. This program is distributed WITHOUT
+# ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+# MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+# AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+#
+#
```

Comment:
I need to study how to do something like: 

```cpp
bind(sourceModuleRFile, "SessionCpp.R")
```

somewhere

Reply:
The general idea is:

1. A pair of files `SessionCpp.hpp` and `SessionCpp.cpp` should be created,
2. A method with signature `core::Error initialize()` should be defined and exported, in some module-specific namespace,
2. That initialize method should use `bind(sourceModuleRFile, "SessionCpp.R")` to ensure that script is sourced on startup,
2. The module's `initialize` method should be called in `SessionMain`, somewhere around here: https://github.com/rstudio/rstudio/blob/main/src/cpp/session/SessionMain.cpp#L543-L628,

Reply:
Thanks. I'll use this and take inspiration from other modules.